server.ts

```diff
@@ -0,0 +1,139 @@
+import express from "express";
+import cors from "cors";
+import { useState } from "react";
+
+const app = express();
+
+// add json handling
+app.use(express.json());
+
+app.use(cors())
+
+const initialBoard = ['', '', '', '', '', '', '', '', '']
+
+
+let games: Game[] = [{
+    id: '123123', data: {
+        board: initialBoard,
```

Comment:
```suggestion
        board: structuredClone(initialBoard),
```

By taking a structured clone of initialBoard, you can avoid any surprises later if you want to refer back to initialBoard.